src/overlays/actors/ovl_En_Ma_Yto/z_en_ma_yto.h

```diff
@@ -2,14 +2,72 @@
 #define Z_EN_MA_YTO_H
 
 #include <global.h>
+#include "overlays/actors/ovl_En_Ma_Yts/z_en_ma_yts.h"
 
 struct EnMaYto;
 
+typedef void (*EnMaYtoActionFunc)(struct EnMaYto*, GlobalContext*);
+
+// Limbs of OBJECT_MA2
+typedef enum {
+    /* 00 */ MA2_LIMB_NONE,
+    /* 01 */ MA2_LIMB_ROOT,
+    /* 02 */ MA2_LIMB_TORSO,
+    /* 03 */ MA2_LIMB_ARM_LEFT,
+    /* 04 */ MA2_LIMB_FOREARM_LEFT,
+    /* 05 */ MA2_LIMB_HAND_LEFT,
+    /* 06 */ MA2_LIMB_ARM_RIGHT,
+    /* 07 */ MA2_LIMB_FOREARM_RIGHT,
+    /* 08 */ MA2_LIMB_HAND_RIGHT,
+    /* 09 */ MA2_LIMB_HEAD,
+    /* 10 */ MA2_LIMB_HAIR,
+    /* 11 */ MA2_LIMB_PELVIS,
+    /* 12 */ MA2_LIMB_TIGHT_LEFT,
+    /* 13 */ MA2_LIMB_LEG_LEFT,
+    /* 14 */ MA2_LIMB_FOOT_LEFT,
+    /* 15 */ MA2_LIMB_TIGHT_RIGHT,
+    /* 16 */ MA2_LIMB_LEG_RIGHT,
+    /* 17 */ MA2_LIMB_FOOT_RIGHT,
+    /* 18 */ MA2_LIMB_DRESS_0,
+    /* 19 */ MA2_LIMB_DRESS_1,
+    /* 20 */ MA2_LIMB_DRESS_2,
+    /* 21 */ MA2_LIMB_DRESS_3,
+    /* 22 */ MA2_LIMB_MAX
+} ObjectMa2Limbs;
+
 typedef struct EnMaYto {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x1E0];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ EnMaYtoActionFunc actionFunc;
+    /* 0x18C */ ColliderCylinder collider;
+    /* 0x1D8 */ struct_800BD888_arg1 unk_1D8;
+    /* 0x200 */ s32 unk_200; // unused
```

Comment:
```suggestion
    /* 0x200 */ s32 unk200; // unused
```
I just wanted to highlight the discussion on whether we should have underscores between unk and numbers. OoT always puts them but MM/Kenix has leaned towards no underscore. I'm fine with either, just thought I'd get the discussion going